lib/src/converter/protobuf_converter.dart

```diff
@@ -1,5 +1,6 @@
 import 'dart:typed_data';
 
+import 'package:flutter_reactive_ble/flutter_reactive_ble.dart';
```

Comment:
Nope, don't import the whole library from inside the library.

Reply:
wow where did that come from?

Reply:
It's pretty easy to end up with using the suggestions from an IDE.